lesson7/index.js

```diff
@@ -0,0 +1,40 @@
+const ul = document.getElementById('js-ul');
+
+function getData() {
+  const data = [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+  ]
+  return data;
+}
+
+function createList() {
+  const LoadingImage = document.createElement('img');
+  LoadingImage.src = "./loading-circle.gif";
+  ul.appendChild(LoadingImage);
```

Comment:
Thanks for the review!

Yes...
I thought it would be better to make it a function considering that it will be used all over.
I fixed it.

Reply:
Thank you!

[IMO]
It might be better to call `loading` outside of the `createList` becuse it's not a part for creating list.
I should've said first... I'm sorry about that.

And you might forget to delete `console.log(lists)` inside of then method. If it's on purpose, never mind!

Reply:
Thank you!
I figured based on this advice...
I want to show loading while I get the data, so
I modified the function name to getData, which means to bring in the data.
What do you think?

I forgot to delete the console.log.
Thank you for letting me know✨

Reply:
This time, it sounds good to me!
I agree with your idea becouse it was just not much for the name `createList`.
But now it's like a one of process of getting data.

Thanks for your time :) It's approve!

Reply:
Thank you for your time!
I felt it was important to make sure the name of the function matched the process and was clear.